Improve upgrade documentation (#855)
digital-asset/docs.daml.com: docs/2.9.5/docs/canton/usermanual/upgrading.rst

    Upgrading requires care and preparation.
      * **Please back up your data before any upgrade.**

>         Note that :ref:`order of the backups <order-of-backups>` is important

Reviewer: ```suggestion
        Note that the :ref:`order of the backups <order-of-backups>` is important
```

      * **Please test your upgrade thoroughly before attempting to upgrade your production system.**

      * **Once a migration has started it cannot be aborted or rolled back.**

>         In case of issue, you need to restore the old version using the backups.

Reviewer: ```suggestion
        If you encounter an issue, you must restore the old version using the backups.
```
